cloudinit/net/dhcp.py

```diff
@@ -29,6 +29,8 @@
 LOG = logging.getLogger(__name__)
 
 NETWORKD_LEASES_DIR = "/run/systemd/netif/leases"
+DHCLIENT_FALLBACK_LEASE_DIR = "/var/lib/dhclient"
+DHCLIENT_FALLBACK_LEASE_REGEX = r".+\.leases*$"  # smth.lease or smth.leases
```

Comment:
I think you meant `.+\.lease*$` to match `.lease` or `.leases`, right?

Also we typically avoid inline comments in this codebase as a style rule, fyi.

Reply:
well, `s*` means 0 or more occurrences of character `s` ;) So it will match '.lease', '.leases', '.leasessss'.

I moved the inline comment.

Reply:
shouldn't it just be `\.leases?$` ?

Reply:
Indeed, I totally forgot about the `?` quantifier. Thanks, fixed that.